hoodie-hadoop-mr/pom.xml

```diff
@@ -118,11 +127,6 @@
   </dependencies>
 
   <build>
-    <resources>
```

Comment:
why remove this? this contains the license/notice

Reply:
I think this was a mistake.

Reply:
n p. could you revert and update the PR